### PR TITLE
docs: add install reference for cloud trace exporter (opentelemetry)

### PR DESCRIPTION
**Pull request by cnnradams**
The example uses the Cloud Trace exporter, but there is no `pip install` mentioned for it in the example.

\+ misc doc fixes

### CODE REVIEW

**Comment by tseaver** [comment]
Likely needs to have PR #124 merged before the main Kokoro build will pass.

**Comment by hengfengli** [comment]
@larkee Could you please help to rebase this PR? I can't do it because the associated email is not my google address.

**Comment by larkee** [comment]
> @larkee ~Could you please help to rebase this PR? I can't do it because the associated email is not my google address.~
> 
> Do you have any idea why `Kokoro docs-presubmit ` fails? I tried to check the invocation log, but it is empty.
> 
> I think the failed test is not caused by this change. It happens in another PR as well: #129.

The `docs-presubmit` is a new CI test that is relying on files that don't yet exist. I'm working on fixing this in #126. I'll prioritize that to unblock this change :+1:

File: docs/opentelemetry-tracing.rst [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 
 .. code-block:: sh
 
     pip install opentelemetry-api opentelemetry-sdk opentelemetry-instrumentation
 
-We also need to tell OpenTelemetry which exporter to use. For example, to export python-spanner traces to `Cloud Tracing <https://cloud.google.com/trace>`_, add the following lines to your application:
+    # [Optional] Installs the cloud monitoring exporter, however you can use any exporter of your choice
+    pip install opentelemetry-exporter-google-cloud
+
+We also need to tell OpenTelemetry which exporter to use. To export spanner traces to `Cloud Tracing <https://cloud.google.com/trace>`_, add the following lines to your application:
```
Comment (larkee) [inline docs/opentelemetry-tracing.rst:16]:
```suggestion
We also need to tell OpenTelemetry which exporter to use. To export Spanner traces to `Cloud Tracing <https://cloud.google.com/trace>`_, add the following lines to your application:
```

**Pull request merged**